src/parser/lexer.mll

```diff
@@ -609,13 +611,13 @@ and code_span buffer nesting_level start_offset input = parse
     { Buffer.add_char buffer c;
       code_span buffer nesting_level start_offset input lexbuf }
 
-  | newline newline
+  | newline whitespace+ as sp
     { warning
         input
         (Parse_error.not_allowed
-          ~what:(Token.describe (`Blank_line "\n\n"))
+          ~what:(Token.describe (`Blank_line sp))
           ~in_what:(Token.describe (`Code_span "")));
-      Buffer.add_char buffer '\n';
+      Buffer.add_char buffer ' ';
       code_span buffer nesting_level start_offset input lexbuf }
```

Comment:
Two newlines are still not allowed. What is allowed is one newline followed by some space (all of that being counted as one space).

Reply:
This new case subsumes the old case. I don't have a good answer to how many spaces should correspond "\n\n" (or more linebreaks), 1 space for every case is the easy answer.
I'm in favor of removing the warning as well.

Reply:
After thinking about this a little bit, I think that whitespace including two newlines should emit a warning. Indeed, code spans are inline elements and we only allow newlines to break long lines.

So the following are equivalent (using `␣` as space to make it more visible):
```
[a␣␣␣b]
[a␣␣
b]
[a␣␣
␣␣␣␣␣b]
```

and the following is forbidden:
```
[a

b]
```

This is closer to what commonmark does (two newlines separate in two paragraph _before_ the inlines inside the paragraph are parsed).